extensions/etcdsnapshot/etcdsnapshot.go

```diff
@@ -260,18 +267,39 @@ func RestoreRKE1Snapshot(client *rancher.Client, clusterName string, snapshotRes
 }
 
 // RestoreRKE2K3SSnapshot is a helper function to restore a snapshot on an RKE2 or k3s cluster. Returns error if any.
-func RestoreRKE2K3SSnapshot(client *rancher.Client, clusterName string, snapshotRestore *rkev1.ETCDSnapshotRestore, initialControlPlaneValue, initialWorkerValue string) error {
+func RestoreRKE2K3SSnapshot(client *rancher.Client, snapshotRestore *rkev1.ETCDSnapshotRestore, clusterName string) error {
```

Comment:
I just want to be sure; the controlPlaneValue and workerValue are still being set on the cluster before the restore, correct?

Reply:
Correct.

Reply:
I have to put that back now considering the changes that were requested. I'll put a new change.